sbin/send_email.py

```diff
@@ -98,7 +98,7 @@ def mark_sent(email_id, db):
 
 def construct_mail_string(send_to, subject, body):
     """Format an email string."""
-    return "TO:%s\nFrom: %s\nSubject:  %s \n\n\n %s \n\n" % (
+    return "Content-Type: text/plain; charset=utf-8\nTO:%s\nFrom: %s\nSubject:  %s \n\n\n %s \n\n" % (
         send_to, EMAIL_SENDER, subject, body)
```

Comment:
Let's make this a bit more readable and easier to parse:

```
headers = [
    ('Content-Type', 'text/plain; charset=utf-8'),
    ('TO', send_to),
    ('From', EMAIL_SENDER),
    ('Subject', subject)
]
msg = ''
for header in headers:
    msg += "{}: {}\n".format(*header)
msg += "\n\n{}\n\n".format(body)
```

This will also fix the error in Travis on linting this file.